homeassistant/components/modbus/switch.py

```diff
@@ -112,15 +111,14 @@
     add_entities(switches)
 
 
-class ModbusCoilSwitch(ToggleEntity, RestoreEntity):
-    """Representation of a Modbus coil switch."""
+class ModbusSwitch(ToggleEntity, RestoreEntity):
```

Comment:
more logical name.

Reply:
`ModbusSwitch` is a base class, which is parent for `ModbusCoilSwitch` and `ModbusRegisterSwitch`. Before, `ModbusRegisterSwitch` inherited from the `ModbusCoilSwitch`, and had useless attributes like `self._coil`, etc.

The name `ModbusSwitch` makes sense to me. However, I could name it `ModbusBaseSwitch` or `ModbusAbstractSwitch`. Just let me know.

Reply:
Since it is a base class the name should reflect it, modbusBaseSwitch sounds good.

Reply:
Renamed.